internal/page/donor/upload_evidence.go

```diff
@@ -118,6 +118,16 @@ func UploadEvidence(tmpl template.Template, payer Payer, documentStore DocumentS
 					}
 
 				case "pay":
+					if len(documents.NotScanned()) > 0 {
+						logger.Print("attempt to pay with unscanned documents on lpa:", lpa.UID)
+						data.Errors = validation.With("upload", validation.CustomError{Label: "errorGenericUploadProblem"})
+						return tmpl(w, data)
+					}
+
+					if err := documentStore.Submit(r.Context(), lpa, documents); err != nil {
+						return err
+					}
+
```

Comment:
Have we thought about how we handle people not paying or being unable to pay? Might be worth talking through with vega to see how they are expecting case workers to work evidence provided cases where payments have not been successful.

Reply:
It's a thing for us too since we only have the happy path for payment confirmation

Reply:
Cool lets leave as is for now and worry about this in a future PR 👍